roxctl/netpol/generate/generate.go

```diff
@@ -0,0 +1,23 @@
+package generate
+
+import (
+	"github.com/np-guard/cluster-topology-analyzer/pkg/controller"
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/pkg/protoconv/networkpolicy"
+)
+
+func (cmd *netpolGenerateCommand) generateNetpol() error {
+	recommendedNetpols, err := controller.PoliciesFromFolderPath(cmd.folderPath)
+	if err != nil {
+		return errors.Errorf("Error synthesizing policies from folder: %v", err)
```

Comment:
You should wrap errors to preserve the context.
```suggestion
		return errors.Wrap(err, "error synthesizing policies from folder")
```

If you use `Errorf` you're creating a new error with a string that contains the other error message. By wrapping the errors you preserve the error chain which can be unwrapped one by one. Plus it allows for convenience functions like `error.Is` to check for the presence of an error in the chain.

Reply:
Thanks for the explanation, changed to Wrap!